src/config/generalconf.cpp

```diff
@@ -51,7 +51,7 @@ GeneralConf::GeneralConf(QWidget* parent)
     updateComponents();
 }
 
-void GeneralConf::updateComponents()
+void GeneralConf::_updateComponents(bool allowEmptySavePath)
```

Comment:
Small detail but maybe this should have a more descriptive name.

Reply:
This is the best I could come up with. My reasoning was that it should have the same name as `updateComponents` because it does the same thing except for a small tweak. And the underscore indicates that it is a helper function. Your suggestions are welcome.